src/components/Counter/Counter.css

```diff
@@ -0,0 +1,14 @@
+@value breakpoint, fontstack from 'styles/variables.css';
+@value black from theme(colors);
+
+.root {
+  color: black;
+  font-family: fontstack, Arial, sans-serif;
+  font-size: 30px;
+}
+
+@media breakpoint {
+  .root {
+    color: #f00;
```

Comment:
Please if possible use `rgb(..)` or `hsl(..)` unless it goes against project team decision